src/screens/Job/generateJobDefinition.ts

```diff
@@ -242,6 +242,28 @@ export const generateJobDefinition = (
       }
 
       break
+    case 'BlockHeaderFeederSpec':
+      values = {
+        ...extractJobFields(job),
+        ...extractSpecFields(
+          job.spec,
+          'coordinatorV1Address',
+          'coordinatorV2Address',
+          'waitBlocks',
+          'lookbackBlocks',
+          'blockhashStoreAddress',
+          'batchBlockhashStoreAddress',
+          'pollPeriod',
+          'runTimeout',
+          'evmChainID',
+          'fromAddresses',
```

Comment:
Is it still fromAddresses? I thought it's `fromAddress` now (singular)?

Reply:
yeah it's still `fromAddresses` . For BHF, we _can_ specify multiple sending keys but only one will be used for the entirety of a single run (i.e. one single key is used for all the batch stores in a single run).